src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx

```diff
@@ -17,6 +17,7 @@ export interface MapPartygoerOverlayProps {
   myUserUid: string;
   totalRows: number;
   totalColumns: number;
+  userPersistentReaction?: UserPersistentReactionType;
```

Comment:
Why is `userPersistentReaction` props drilled through:

- `src/components/templates/PartyMap/components/Map/hooks/usePartygoersOverlay.tsx`
- `src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx`
- `src/components/molecules/UserProfilePicture/UserProfilePicture.tsx`

It looks like `usePartygoersOverlay` has the following:

```
const isUserPersistentReaction =
  peopleRequesting?.some((user) => user.id === partygoer.id) || false;
```

I'm not sure that this should be happening at this level of things..?